docs/src/getting_started.rst

```diff
@@ -51,8 +51,9 @@ Setting up your own project
 
   * `Git <https://git-scm.com/downloads>`_
 
+2. If you use Mac or Linux, open your terminal. If you use Windows, please open Windows Powershell. Pleae make sure that the path to your future project directory does not include any spaces or special characters. Further, you need to make sure that every program that is used in your project (Stata, R, Matlab, Latex Distributions, Anaconda) can be found on your *PATH*. That is, these need to be accessible from your shell. This is not always automatically true, in particular on Windows. In Windows, one has to oftentimes add the programs manually to the PATH environmental variable in the Advanced System Settings. How to exactly do that on windows see `here <https://www.computerhope.com/issues/ch000549.htm>`_ and on Mac see `here <http://hmgaudecker.github.io/econ-python-environment/paths.html#macos-adjust-slightly-for-linux>`_
```

Comment:
Let us please try to be more verbose here and a bit more checklist style. Suggestion, after "open Windows Powershell":

Navigate to the parent folder of your future directory. Please ensure that 

1. Typing `pwd` reveals that there are no spaces or special characters (Umlaute, ... [see what I had linked for all examples]) in the path. Else change to C:\ whatever and create your project there.
2. Typing `git status` starts the programme git  and it tells you that there is no repository present, e.g., something like: 
     ```
fatal: not a git repository (or any of the parent directories): .git
     ```
3. Typing `python` starts the programme Python from Anaconda (exit with Ctrl+D).
4. Similarly, every programme ... [should we do more examples? pdflatex? Give the list for stata / r / matlab from the run_xx_scripts?]

If any of 2.-4. fails, you will need to adjust the path... [continue with above, please link something on SO or so for MacOS or create an FAQ item, I want to take that site down asap]

If you have not used the Powershell much, looking at Tips & Tricks ... might be worthwhile .

Reply:
5. Typing `atom` should start the text editor `Atom <https://....>`__ If you do not want to use it you will need to adjust some commands for yourself.

Reply:
Then we should mention somewhere before that they need an editor, are there no system specific build in editors that we can access through the command line, we might be creating a lot of overhead here.

Reply:
Yes. Sorry I missed that, too many changes at various places :-)

Please add as a fourth bullet point under 1. Required programmes.

Reply:
Oh, if people do not use a reasonable editor, they should start doing so immediately. If they do, changing 'atom' for 'subl' or 'vim' will be obvious to them. So this is a feature rather than unnecessary overhead.